### ISSUE TITLE
Can we receive data on an RPi instead of chromecast ? 

### ISSUE THREAD

**Issue by SainiVinod** [acidhax/chromecast-audio-stream#62]
This is less of an issue, and more of a question, I am trying to make a RPi based such server, where I can receive audio. Can we use this to receive data on an RPi 2 or 3. Thank you.

**Comment by matbeedotcom** [comment]
Yeah, it's possible, but a custom implementation. I have a couple PiZero's and thought about implementing it there, too. Are you already running a UPNP/DLNA server?

**Comment by mcprat** [comment]
First of all, you must have something running on the PI. BubbleUPnP Server detects Chromecast and other things and can turn it into a DLNA Renderer but it depends on what software runs on it (not necessary if its already running as a client). After that, casting to it is easy, but maybe not with this program. A better example program for DLNA would be SWYH.

Heres a guide I found for the first and most difficult part of it.
http://www.instructables.com/id/Raspberry-Pi-DLNA-serverrenderer/
